mautic/mautic
# Fix Go To Webinar email validation

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:

If form with GTW action has different email form field alias like `email` it make issue and validation message is not display in form. This PR fixed it


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Setup GTW
3. Create form with GTW subscription action. Email form field name should be `Email profesional` for example
4. Try submit / register to GTW twice.
5. See second time you don't see validation error , event If we expect see validation error Email already registered to webinar




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11757"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
